webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -363,6 +363,10 @@ const TableActionBar: React.FC<Props> = ({
               </Button>
             </Dropdown>
           )}
+          <span className={css.expNum}>
+            {Loadable.getOrElse(0, total).toLocaleString()} experiment
+            {Loadable.getOrElse(0, total) > 1 && 's'}
```

Comment:
**nit refactor:** rewrite to only run `Loadable.getOrElse(0, total)` once?

Reply:
fixed